components/director/internal/domain/runtime/converter.go

```diff
@@ -1,4 +1,82 @@
 package runtime
 
-type Converter struct {
+import (
+	"github.com/kyma-incubator/compass/components/director/internal/graphql"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+)
+
+type Converter struct{}
+
+func (c *Converter) ToGraphQL(in *model.Runtime) *graphql.Runtime {
+	if in == nil {
+		return nil
+	}
+
+	return &graphql.Runtime{
+		ID:          in.ID,
+		Status:      c.statusToGraphQL(in.Status),
+		Name:        in.Name,
+		Description: in.Description,
+		Tenant:      graphql.Tenant(in.Tenant),
+		Annotations: in.Annotations,
+		Labels:      in.Labels,
```

Comment:
what about agentAuth?

Reply:
It will be done in a follow-up task, as this is more complicated topic: #67